documentation/docs/strategies/advanced-mm/add-transaction-costs.md

```diff
@@ -1,18 +1,33 @@
 # Adding Transaction Costs to Prices
 
-Transaction costs can now be added to the price calculation. `fee_pct` refers to the percentage maker fees per order (generally common in Centralized exchanges) while `fixed_fees` refers to the flat fees (generally common in Decentralized exchanges).
+**Updated as of `v0.28.0`**
 
-- The bid order price will be calculated as:
+This feature lets you take into account the transaction cost incurred on the exchange from bid and ask orders. For a bid order, it reduces the bid order price and increases the ask order price to take into account the transaction cost.
```

Comment:
This is a good description, but try to make it clearer:
"For a bid order, it reduces the price by the fee amount. For an ask order, it increases the price by the fee amount."